src/main/java/com/mailchimp/domain/List.java

```diff
@@ -16,13 +18,16 @@
  * @author Micheal Swiggs
  * @author stevensnoeijen
  */
-public class SubscriberList {
+public class List {
```

Comment:
Not to sure about this name. Java already has a `List`. Users will need to start qualifying if using both types in a file.

Reply:
I agree and changed it back